controllers/goharbor/harborcluster/service.go

```diff
@@ -14,23 +13,18 @@ import (
 	"k8s.io/client-go/tools/record"
 )
 
-type ServiceReconciler interface {
-	// Reconcile the dependent service.
-	Reconcile(harborCluster *v1alpha2.HarborCluster) (*lcm.CRStatus, error)
-}
-
 type ServiceGetter interface {
```

Comment:
I think the lcm controllers can be created in the `New` method of the HarborCluster Reconciler struct.

Reply:
ServiceGetter is created in New method